templates/Managerpage.html

```diff
@@ -12,18 +12,28 @@
 <form action="/managerSearch" method="post">
 <center><b>Generate Product orders</b></center>

```

Comment:
This probably won't affect anything, but the `<b>` tag is only supposed to be used as a last resort now in html 5. They suggest to use css instead. Probably won't fix the button though